app/src/main/java/com/nextcloud/talk/controllers/ContactsController.java

```diff
@@ -327,15 +328,19 @@ private void fetchData(boolean forceNew) {
     }
 
     private void prepareViews() {
-        recyclerView.setLayoutManager(new SmoothScrollLinearLayoutManager(getActivity()));
+        LinearLayoutManager layoutManager = new SmoothScrollLinearLayoutManager(getActivity());
```

Comment:
Memory vs readability, like mentioned. But it's good, no worries. Thanks for the contribution, and keep them coming!

Reply:
Done!

Reply:
via https://github.com/nextcloud/talk-android/pull/12/commits/3a6d2c752d1958c356f12559ee7f7e8b4f46e53c